foodsaving/groups/templates/user_removed_from_group.subject.jinja2

```diff
@@ -0,0 +1 @@
+[{{ group_name }}] {% trans %}Group removal{% endtrans %}
```

Comment:
"Group removal" sounds like the group is getting removed... how about "Removal from group"?